notebooks/binary_predicates.ipynb

```diff
@@ -0,0 +1,1157 @@
+{
```

Comment:
<p>This section seems better to exist in code comments, not in notebook...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>